docs/reference/technology/connectivity/connectivity.md

```diff
@@ -98,21 +103,21 @@ The [LoRa](lora-tech.html) section and [LoRa tutorial](../tutorials/LoRa-tutoria
 
 <span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/n_mark.png)<span>NFC</span></span>
 
-Near-field communication (NFC) is a short range (few centimeters) wireless technology standard for personal area networks. Typical uses of NFC are commissioning, easy sharing of small content and Bluetooth connection initiation/out-of-band pairing.
+Near-field communication (NFC) is a short range (few centimeters) wireless technology standard for personal area networks. Typical uses of NFC are commissioning, easy sharing of small content, and Bluetooth connection initiation/out-of-band pairing.
 
 To learn how to use NFC with Mbed OS, please refer to the [Mbed OS NFC overview](../apis/nfc.html).
 
 ### NB-IoT cellular
 
-Non-IP Data Delivery (NIDD) is a new feature for communication over NB-IoT. It is enabled by Control Plane cellular IoT EPS optimization and meant to provide improved support of small data transfer. It does this by transporting user data over the control channel, thus reducing the total number of control plane messages when handling a short data transaction.
+Non-IP Data Delivery (NIDD) is a new feature for communication over NB-IoT. It is enabled by Control Plane cellular IoT EPS optimization, to provide improved support for small data transfer. It transports user data over the control channel, thus reducing the total number of control plane messages when handling a short data transaction.
 
 To learn how to use this feature with Mbed OS, please refer to [CellularNonIPSocket](../apis/non-ip-cellular-socket.html).
 
 ### Memory needs for Pelion-connected devices
 
-Mbed OS's baseline memory footprint, without Pelion connectivity, is 2.8kb of RAM and 8.2kb of flash. When you add connectivity and full functionality, the footprint grows:
+Mbed OS's baseline memory footprint, without Pelion connectivity, is 2.8 KB RAM and 8.2 KB flash. When you add connectivity and full functionality, the footprint grows:
 
```

Comment:
@teetak01, can you please check the numbers here.

Reply:
I cannot say on the Mbed OS numbers (which look to be unrealistically low, especially for flash).

For client, I would just link here: https://www.pelion.com/docs/device-management/current/cloud-requirements/requirements-for-production-devices.html